services/libs/types/src/enums/organizations.ts

```diff
@@ -36,8 +36,9 @@ export enum OrganizationIdentityType {
 
 export enum OrganizationAttributeSource {
   CUSTOM = 'custom',
-  PDL = 'peopledatalabs',
-  EMAIL = 'email',
-  GITHUB = 'github',
+  ENRICHMENT_INTERNAL_API = 'enrichment-internal-api',
```

Comment:
NIT: I think this naming is too vague - Internal API of what? There can be multiple internal APIs. If this is from LFX, we should add it to the naming or give it a more descriptive name